CommunityToolkit.Net.Authentication/MockProvider.cs

```diff
@@ -0,0 +1,60 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Net.Http;
+using System.Threading.Tasks;
+using CommunityToolkit.Net.Authentication.Extensions;
+
+namespace CommunityToolkit.Net.Authentication
+{
+    /// <summary>
+    /// Provider to connect to the example data set for Microsoft Graph. Useful for prototyping and samples.
+    /// </summary>
+    public class MockProvider : BaseProvider
```

Comment:
Looks like we missed `git mv` here? Unfortunately don't think there's an easy way to rectify?

Reply:
Lame, Git didn't detect the move. All I did was extend the BaseProvider and remove all of the redundant methods. Otherwise it's the same logically.